vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -74,6 +82,8 @@ func init() {
 // available throughout Kubernetes binaries.
 var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureSpec{
 	StreamingProxyRedirects: {Default: true, PreRelease: utilfeature.Beta},
+	// CVE-2018-1002102: Unvalidated redirect: enable ValidateProxyRedirects by default
+	ValidateProxyRedirects:  {Default: true, PreRelease: utilfeature.Alpha},
```

Comment:
would but the default change into its own commit. It's not really from the upstream commit.